components/brave_rewards/resources/extension/brave_rewards/manifest.json

```diff
@@ -25,6 +25,22 @@
       "256": "img/bat-256.png"
     }
   },
+  "content_scripts": [
+    {
+      "matches": [
+        "https://*.twitter.com/*"
+      ],
+      "css": [
+        "content.css"
```

Comment:
I suggest we don't add a css file to the `content_scripts` since, unlike JS, I believe this is added to the global document styles. Instead, especially since these styles are minimal, perhaps we add them via JS, and ideally within a shadow DOM so that nothing is affected by the document's stylesheet.

Reply:
Great point, I didn't realize that. Will fix, thank you.